data/korath/korath missions.txt

```diff
@@ -61,3 +61,353 @@ mission "Discovered Korath Space"
+			`	The Korath who had spoken to you points up to the sky and says, "<planet>. <planet>," while gesturing to itself and the two Korath. It presses 500 credits into your hands and continues to mutter in its own language. <planet> is the name of the Quarg ringworld in Korath space. They must want transportation to it. 500 credits is shockingly low for any transport job, but based on their look they can't afford to pay you any more. Do you want to take them to <planet>?`
+			choice
+				`	(Yes.)`
+				`	(Yes, and give their credits back to them.)`
+					goto forfree
+				`	(No.)`
+					goto heartless
+			`	You pocket the credits and gesture for them to follow to your ship. When the Korath realizes you've agreed, it sighs in relief and rapidly gestures in gratitude. You show them to their bunks and the shower - they're covered in soot.`
+				accept
+			label forfree
+			`	The Korath's shoulders slump when you hand back the credits, but its disappointment turns to surprise when you gesture to your ship. When it realizes you're trying to refuse the money, it puffs its chest out and pushes the credits into your hands again. It won't take no for an answer, so you pocket the credits and take them to your ship. You show them to their bunks and the shower - they're covered in soot.`
+			accept
+			label heartless
+			`	You give the credits back to the Korath, and its shoulders slump. The trio wanders off, all three coughing, until the smog envelops them. You can't help but feel a little bad.`
+				decline
+	on accept
+		payment 500
+	on complete
+		conversation
+			`You enjoy the new company on your ship. The child has taught you a game where you take turns throwing a ball against the wall. You spend an hour or two playing with it in the cargo hold.`
+			`	When you arrive, you decide to accompany the family into the spaceport. Waiting near the docking bay are two more Korath, one an adult and the other elderly, and the three you've transported rush to meet them. After the happy reunion has died down, the Korath who had originally asked you for transportation turns to a nearby Quarg and chatters off something. The Quarg responds and then turns to you.`
+			`	"This prideful fellow has said he regards my translation skills with reverence and asks I use them. I am inclined to acquiesce his request." This Quarg does not seem to have an especially strong grasp of your language, but you should be able to roughly gather what is being said. "The comrade says, 'I disproportionately thank you for reuniting my familial, in fateful service to our comraderie.' He additionally supplants there was no method in which this gaggle would vicinitied in which absenced your presence. That is all."`
```

Comment:
to me it seems the first sentence isn't nearly as tough to understand as the rest of what the Quarg says; since it's after that one and another sentence that the game tells you that `This Quarg does not seeem to have an especially strong grasp of your language`, maybe make the first one a bit more convoluted like the rest
//
when googling `comraderie` I only get that word from Urban Dictionary, so pointing it out in case it's not really a word but a typo of camaraderie or something else.